### DISCUSSION TITLE
Create an Account via Google

### DISCUSSION THREAD

**Discussion by Btibert3** [sqlitecloud/discussions#4] [Feature request] [answered]
This could be a great option to support my teaching needs.  It would be nice if students could create an account quickly via their university-associated Google Account.

**Comment by marcobambini** [comment]
Thanks for the feedback @Btibert3.
This is something we are already working on.
I'll let you know as soon as it will be implemented.

**Comment by Gioee** [comment, marked as answer]
@Btibert3 Now we do have Google and GitHub authentication in our dashboard.
